{"sections":[],"metadata":{"modules":[{"name":"Auth"}],"title":"Error Implementations","role":"collectionGroup"},"topicSections":[{"identifiers":["doc:\/\/Auth\/documentation\/Auth\/AuthError\/localizedDescription"],"title":"Instance Properties","generated":true}],"hierarchy":{"paths":[["doc:\/\/Auth\/documentation\/Auth","doc:\/\/Auth\/documentation\/Auth\/AuthError"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/Auth\/documentation\/Auth\/AuthError\/Error-Implementations"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/auth\/autherror\/error-implementations"]}],"kind":"article","schemaVersion":{"patch":0,"major":0,"minor":3},"references":{"doc://Auth/documentation/Auth":{"kind":"symbol","url":"\/documentation\/auth","role":"collection","title":"Auth","identifier":"doc:\/\/Auth\/documentation\/Auth","type":"topic","abstract":[]},"doc://Auth/documentation/Auth/AuthError":{"identifier":"doc:\/\/Auth\/documentation\/Auth\/AuthError","abstract":[{"text":"Authentication error","type":"text"}],"title":"AuthError","kind":"symbol","url":"\/documentation\/auth\/autherror","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"AuthError","kind":"identifier"}],"navigatorTitle":[{"text":"AuthError","kind":"identifier"}],"type":"topic","role":"symbol"},"doc://Auth/documentation/Auth/AuthError/localizedDescription":{"abstract":[],"kind":"symbol","type":"topic","identifier":"doc:\/\/Auth\/documentation\/Auth\/AuthError\/localizedDescription","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"localizedDescription","kind":"identifier"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"}],"url":"\/documentation\/auth\/autherror\/localizeddescription","title":"localizedDescription","role":"symbol"}}}